inc/Engine/Admin/Settings/Page.php

```diff
@@ -826,6 +826,34 @@ private function assets_section() {
 					'default'           => 1,
 					'sanitize_callback' => 'sanitize_checkbox',
 				],
+				'delay_js'           => [
+					'container_class'   => [
+						'wpr-isParent',
+					],
+					'type'              => 'checkbox',
+					'label'             => __( 'Delay JavaScript execution', 'rocket' ),
+					'description'       => sprintf( __( 'Improves initial loading time by delaying the loading of JavaScript files.', 'rocket' ) ),
```

Comment:
```suggestion
					'description'       => __( 'Improves initial loading time by delaying the loading of JavaScript files.', 'rocket' ),
```

The `sprintf` is not needed in this case